feat: [DM-4719] Add HTTP proxy support (#3460)
SoftwareAG/c8y-docs: content/change-logs/device-management/cumulocity-10-21-06-0-OPC-UA-HTTP-Proxy-Support.md

---

> date: 2023-12-06T11:45:06.705Z

Reviewer: ```suggestion
date: 
```

> title: HTTP Proxy support for the OPC UA Gateway

Reviewer 2: ```suggestion
title: HTTP proxy support for the OPC UA gateway
```

change_type:
  - value: change-QHu1GdukP
    label: Feature
product_area: Device Management
component:
  - value: component-Tf05_KQ-B
    label: OPC UA
build_artifact:
  - value: tc-QHwMfWtBk7

>     label: cumulocity

Reviewer: ```suggestion
  - value: tc-MLn0oFRX-
    label: opcua
```

ticket: DM-4719
version: 10.21.6.0
---

> Added support for operating behind an HTTP proxy. This enhancement allows the Cumulocity OPC UA Gateway to be used in environments where an HTTP proxy is required, improving flexibility and compatibility with various network configurations.

Reviewer 2: ```suggestion
Support has been added for operating behind an HTTP proxy. This enhancement allows the {{< product-c8y-iot >}} OPC UA gateway to be used in environments where an HTTP proxy is required, improving flexibility and compatibility with various network configurations.
```
